source/opt/local_access_chain_convert_pass.cpp

```diff
@@ -139,7 +139,17 @@ bool LocalAccessChainConvertPass::IsConstantIndexAccessChain(
 bool LocalAccessChainConvertPass::HasOnlySupportedRefs(uint32_t ptrId) {
   if (supported_ref_ptrs_.find(ptrId) != supported_ref_ptrs_.end()) return true;
   analysis::UseList* uses = get_def_use_mgr()->GetUses(ptrId);
-  assert(uses != nullptr);
+
+  if (!uses) {
+    // This is a variable (or access chain to a variable) that has no uses.
+    // We won't encounter loads or stores for this <result-id> per se, but
+    // this <result-id> may be derived from some other variable (or access
+    // chain). Return true here can unblock the access chain conversion of
+    // the root variable. This particular <result-id> won't be touched and
+    // can be handled in dead code elimination.
+    return true;
+  }
+
```

Comment:
This should not be needed after @alan-baker PR #987 is in.  @alan-baker could you confirm?

Reply:
Agree.  #987 changes the API to provide a ForEachUse or ForEachUser and you can't get nullptr at all.

Reply:
Oh, great! That would be better than this ad-hoc fix.